src/install/linux.md

```diff
@@ -1,9 +1,7 @@
 ---
-layout: default
-title: "Installing Dart on Linux"
-description: "Installing and updating the Dart SDK on Linux with apt-get, a Debian package, and compiling from source."
+title: Installing Dart on Linux
+description: Installing and updating the Dart SDK on Linux with apt-get, a Debian package, and compiling from source.
```

Comment:
Did you consider setting a short-title of Linux?

(Not sure it's better... but it's a little more readable, I suppose.)

Reply:
None of the other OS install pages had short titles. Maybe:
> short-title: Linux Install

and similarly for other OS's?

Reply:
I was thinking that they'd all have short titles with just the OS name, so you'd see:
* Install > Linux
* Install > Mac
* Install > Windows